client-ts/signalr/src/HttpConnection.ts

```diff
@@ -42,7 +42,10 @@ export class HttpConnection implements IConnection {
     constructor(url: string, options: IHttpConnectionOptions = {}) {
         this.logger = LoggerFactory.createLogger(options.logger);
         this.baseUrl = this.resolveUrl(url);
+
         options = options || {};
+        options.accessTokenFactory = options.accessTokenFactory || (() => null);
```

Comment:
Would it be easier to just do a null check on the `accessTokenFactory` before creating and checking the token. It gets rid of some of the logic

Reply:
In a couple of places